### DISCUSSION TITLE
Potential issue with accuracy and score advantage from using classic mod / playing on stable impacting usage of standardised scoring

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#25621] [Q&A]
I just set two plays on lazer:
![image](https://github.com/ppy/osu/assets/129146357/9fdd79a2-086d-483d-a6c4-8a25993f2773)

One was set using scorev2 / standardised scoring (the default on lazer), and the other was set using the classic mod. The one with the classic mod was objectively easier to set a high accuracy play on, I would have received one 100 and one 50 on sliders if I were to have submitted that play without the classic mod.

My point being, there is a competitive advantage from using the classic mod compared to standardised scoring (and presumably from setting plays in the stable client once leaderboards are merged). I believe that this has some negative implications which could impact user acceptance of lazer and the leaderboards. Players looking to maximise their score / pp / leaderboard position on a play will only use classic mod or always play on stable because there is no incentive to use standardised scoring as it only makes it harder to set a good score.

One way to remedy this would be to unrank the classic mod and add slider accuracy / standardised scoring to stable (and replace the scorev2 mod with the classic mod). As far as I can tell, the only other way to prevent players from only using the classic mod or only playing on stable would be to change the accuracy and score to reflect scores set on the classic mod to be as they would if they were set using standardised scoring when submitted on leaderboards, but I don't know if that is feasible and may only cause unnecessary confusion.

I'm not sure if this is the right place to put forward this kind of issue, bug report didn't seem appropriate since this isn't a technical problem.

**Comment by peppy** [comment]
Classic mod now has a debuff multiplier.

**Reply by ghost** [reply]
There is still the issue of it (generally) being easier to gain pp using the classic mod / by playing on stable. Slider accuracy just seems to cause so many headaches with the leaderboards and user acceptance, at some point you must wonder if it's worth all the trouble. A score debuff could be seen as unfair since there are players who performed just as well on the map in stable / using classic mod as someone playing with slider acc (especially if the map has fewer or only easier sliders - there is also that with the newer slider changes some maps are actually easier on lazer now). Removing the score debuff seems like a much easier approach, at least for user acceptance.

**Reply by Rekunan** [reply]
There are plans to properly account for this in pp, the main question being how. If you're interested in keeping up with progress I recommend joining the pp dev discord: https://discord.gg/aqPCnXu
